blueocean-git-pipeline/src/test/java/io/jenkins/blueocean/blueocean_git_pipeline/GitUtilsTest.java

```diff
@@ -94,4 +97,25 @@ public void testOperations() throws Exception {
         text = FileUtils.readFileToString(new File(repo.getRoot(), "test.txt"), "utf-8");
         Assert.assertTrue(secondText.equals(text));
     }
+
+    @Test
+    public void testValidatePushAccessFails() throws Exception {
+        User user = login();
+        this.jwtToken = getJwtToken(j.jenkins, user.getId(), user.getId());
+
+        Map resp = new RequestBuilder(baseUrl)
+            .status(200)
+            .get("/organizations/jenkins/user/publickey/").build(Map.class);
+
+        String id = (String)resp.get("id");
+        Assert.assertTrue(id != null);
+
+        resp = put("/organizations/jenkins/scm/git/validate/",
+            ImmutableMap.of(
+                "repositoryUrl", "git@github.com:vivek/test-no-jenkins-file.git",
```

Comment:
if this repo disappears or changes... will it break the test?

Reply:
yes, but it's used in [another test already](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-git-pipeline/src/test/java/io/jenkins/blueocean/blueocean_git_pipeline/GitScmTest.java#L134) @michaelneale

Reply:
err, mebby not. no idea where i got that repo from, copied it from some test somewhere :)